sanic/request.py

```diff
@@ -53,6 +56,38 @@
     from json import loads as json_loads  # type: ignore
 
 
+@dataclasses.dataclass()
```

Comment:
In other locations in the project we use just `@dataclass`. Let's keep it consistent.